tests/unit/features/writer/components/left-sidebar/sidebar-chapter.test.tsx

```
@@ -1,184 +1,200 @@
-  ContextMenuContent: ({ children }: any) => <div data-testid="ctx-content">{children}</div>,
-  ContextMenuItem: ({ children, onClick }: any) => <button onClick={onClick}>{children}</button>,
-  ContextMenuSeparator: () => <hr />,
+	ContextMenu: ({ children }: any) => <div>{children}</div>,
+	ContextMenuTrigger: ({ children }: any) => (
+		<div data-testid="ctx-trigger">{children}</div>
+	),
+	ContextMenuContent: ({ children }: any) => (
+		<div data-testid="ctx-content">{children}</div>
+	),
+	ContextMenuItem: ({ children, onClick }: any) => (
+		<button onClick={onClick}>{children}</button>
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)